webapp/carts/models.py

```diff
@@ -0,0 +1,42 @@
+from django.conf import settings
+from django.db import models
+
+
+User = settings.AUTH_USER_MODEL
+
+
+class Cart(models.Model):
+    #allowed nulls and blanks for future guest checkout
+    user = models.ForeignKey(
+        User,
+        null=True,
+        blank=True)
+
+    book = models.ForeignKey(
+        'book_details.Book',
+        related_name='product')
+
+    total_price = models.DecimalField(
+        'total price of all items',
+        default=0.00,
+        max_digits=100,
+        decimal_places=2)
+
+    updated = models.DateTimeField(
+        auto_now=True)
+
+    timestamp = models.DateTimeField(
```

Comment:
What are these time fields for?
Please add descriptions to your model fields so that they are self documented.

Reply:
Just tracking updates to the cart I felt it was redundant

Reply:
I'm more for keeping things lean; YAGNI and all that.
I'd withdraw my comment if other team members disagree.
Also, the `timestamp` doesn't indicate what it is explicitly a timestamp of; I'd suggest `created` are `creation_time` or something along those lines if that's its goal.